### PR TITLE
add Swing example

### PR DESCRIPTION
**Pull request by MoAlyousef**
This PR adds a Java/swing example

### CODE REVIEW

File: toolkits/Java/swing/bench.dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM debian:stable-slim
+
+CMD java executable.app
```
[the remark this answers is not in the document]
Comment (MoAlyousef) [reply toolkits/Java/swing/bench.dockerfile:3]: It appears so. I tried running an install default-jre step but that failed due to conflicts. 
Invoking java directly seems to work!

[the remark this answers is not in the document]
Comment (MoAlyousef) [reply toolkits/Java/swing/bench.dockerfile:3]: Then again, looking at docker desktop's image, I see:
```
/bin/sh: 1: java: not found
```

I'll try to find another way

[the remark this answers is not in the document]
Comment (MoAlyousef) [reply toolkits/Java/swing/bench.dockerfile:5]: Adding:
```
RUN apt-get update && apt-get install -qq default-jre
```
without the `--no-install-recommends` seems to work.

**Review by IronOxidizer — APPROVED** [review]

**Comment by IronOxidizer** [comment]
Resolved https://github.com/IronOxidizer/gui-toolkit-benchmarks/issues/27

**Comment by IronOxidizer** [comment]
Results published

**Pull request merged**